f5_openstack_agent/lbaasv2/drivers/bigip/l2_service.py

```diff
@@ -202,9 +202,9 @@ def assure_bigip_network(self, bigip, network):
         elif network['provider:network_type'] == 'vxlan':
             network_name = self._assure_device_network_vxlan(
                 network, bigip, network_folder)
-        elif network['provider:network_type'] == 'gre':
-            network_name = self._assure_device_network_gre(
-                network, bigip, network_folder)
+        elif network['provider:network_type'] == 'opflex':
+            raise f5_ex.NetworkNotReady(
```

Comment:
Do we know for sure that the network is not yet ready? Have we inspected the segment_id and found it 'null'?